Sources/AsyncHTTPClient/ConnectionPool/HTTPConnectionPool.swift

```diff
@@ -12,8 +12,114 @@
 //
 //===----------------------------------------------------------------------===//
 
+import NIO
+
 enum HTTPConnectionPool {
-    struct Connection {
+    struct Connection: Equatable {
         typealias ID = Int
+
+        private enum Reference {
+//            case http1_1(HTTP1Connection)
```

Comment:
Can you provide explanations for why all of these are commented out?

Reply:
Yes. The the sad and simple reason is: We don't have a HTTP1Connection yet. All of those will be enabled as soon as the `HTTP1Connection` has landed. Landing the connection box now enables us to already review the ConnectionPool StateMachines.

Reply:
Sorry, I meant in the code. :wink:

Reply:
Damn it. Fixed.